uproot/write/objects/TTree.py

```diff
@@ -21,11 +22,13 @@
 
 class newbranch(object):
 
-    def __init__(self, type, title="", shape=(1,), **options):
+    def __init__(self, type, title="", shape=(1,), dependence=None, **options):
```

Comment:
"dependence" is an odd name, not easily guessed. This matters because users will have to type it directly. How about "lengths"?

Reply:
What do you think about `lengthsbranch` over `lengths`?

Reply:
This still needs to be renamed.

Reply:
Technically, it's a TLeaf that is associated with counting (in the rare case that it's on a multi-leaf branch). We're trying to switch to more snake_case words, like lengths_branch. Also, it's getting long.

Maybe "counter" is clearer than lengths because it sounds like an active word.

Reply:
Okay, I changed it to counter.